### ISSUE TITLE
Update documentation and source type files with optional supported operations

### ISSUE THREAD

**Issue by jcguion** [Vantiq/vantiq-extension-sources#202]
As it is possible to specify the list of supported operations within the source type file, both documentation (README) and existing source type files should be updated with the appropriate example and values. This (optional) property is named 'operations' and the property value is an array of strings, each element of the array defining a supported operation.

**Comment by fhcarter** [comment]
@jcguion What is the list of "supported" operations?  I assume it's 'publish', 'query', and 'notification'?  I assume these are the same ops from ResourceConstants.{NOTIFICATION, PUBLISH, QUERY}?

**Comment by jcguion** [comment]
Yes to both, `publish` `query` and `notification`, and the same ops from ResourceConstants.
